src/ui-manager.js

```diff
@@ -62,8 +149,30 @@ class UIManager {
     this.setConfig(config);
     this._setLocaleTranslations(config);
     setEnv(this.player.env);
+
+    this._resizeWatcher = new playkitUtils.ResizeWatcher();
+    this._resizeWatcher.init(document.getElementById(config.targetId));
+    this._resizeWatcher.addEventListener(CustomEventType.RESIZE, this._onPlayerWrapperResize);
```

Comment:
this is thrown from player, what is the use case to hold two instances? just listen to resize from player